packages/react/src/examples/forms/validation.scss

```diff
@@ -3,14 +3,14 @@
   max-width: 792px;
 
   &__main-title {
-    color: var(--color-bus);
+    color: var(--color-black-90);
```

Comment:
It might be unnecessary to declare titles' colors anymore since they 
use the same color as the body text.

Reply:
Ty, removed it.